db/migrate/20231030184455_create_comments.rb

```diff
@@ -0,0 +1,8 @@
+class CreateComments < ActiveRecord::Migration[6.1]
+  def change
+    create_table :comments, &:timestamps
```

Comment:
Interesting solution! It looks cool! However, in the rails community everyone is trying to adhere to the standard so that other developers understand the code) Well done